roles/rsyslog/defaults/main.yaml

```diff
@@ -977,31 +981,36 @@ rsyslog_conf_viaq_elasticsearch:
                   tls.myprivkey="{{logging_elasticsearch_key}}"
               {% endif %}
               )
-          {% if openshift_logging_use_ops | default(False) %}
-              } else {
-              action(
-                  type="omelasticsearch"
-                  server="{{ elasticsearch_ops_server_host | default('logging-es-ops') }}"
-                  serverport="{{ elasticsearch_ops_server_port | default(9200) }}"
-                  template="viaq_template"
-                  searchIndex="index_template"
-                  dynSearchIndex="on"
-                  searchType="com.redhat.viaq.common"
-                  bulkmode="on"
-                  writeoperation="create"
-                  bulkid="id_template"
-                  dynbulkid="on"
-                  retryfailures="on"
-                  retryruleset="try_es"
-                  usehttps="on"
-              {% if use_omelasticsearch_cert | default(False) %}
-                  tls.cacert="{{logging_elasticsearch_ca_cert}}"
-                  tls.mycert="{{logging_elasticsearch_cert}}"
-                  tls.myprivkey="{{logging_elasticsearch_key}}"
-              {% endif %}
+          {% else %}
+          {% if res.name|default("") == "primary" %}
+              if $.viaq_index_prefix startswith "{{ res.index_prefix }}" then {
+          {% else %}
+              elif $.viaq_index_prefix startswith "{{ res.index_prefix }}" then {
```

Comment:
I don't think rsyslog rainierscript has an `elif` keyword

Reply:
Oops.  Thanks, @richm .  I mixed up with the ansible syntax... :p
Then, all we could do is like this?
```
if $.viaq_index_prefix startswith "projects." then {
}
if $.viaq_index_prefix startswith ".operations." then {
}
...
```